coderd/provisionerdserver/provisionerdserver.go

```diff
@@ -74,6 +74,37 @@ const (
 	StaleInterval = 90 * time.Second
 )
 
+// incrementWorkspaceCreationOutcomesMetric increments the workspace creation
+// outcomes metric for first 'start' builds. This counts regular (non-prebuilt)
+// workspace creation outcomes (success/failure).
+func (s *server) incrementWorkspaceCreationOutcomesMetric(ctx context.Context, workspace database.Workspace, workspaceBuild database.WorkspaceBuild, job database.ProvisionerJob) {
+	if workspaceBuild.BuildNumber == 1 &&
+		workspaceBuild.Transition == database.WorkspaceTransitionStart &&
+		workspaceBuild.InitiatorID != database.PrebuildsSystemUserID {
```

Comment:
We already do similar checks for the timing metrics `coderd_workspace_creation_duration_seconds` and `coderd_prebuilt_workspace_claim_duration_seconds` on `completeWorkspaceBuildJob`: https://github.com/coder/coder/blob/1aca85ae3acceea8c89949b8bda301be43899826/coderd/provisionerdserver/provisionerdserver.go#L2393
As well as, getting the presetID from the database, so maybe we could update all metrics in the same place?

Reply:
I'll have a look 👍